charts/monitoring-rules/templates/podmonitor-filebeat.yaml

```diff
@@ -0,0 +1,17 @@
+{{ if and .Values.logging.enabled .Values.logging.podMonitor }}
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: {{ include "monitoring-rules.fullname" . }}-logging-filebeat
+  namespace: logging
```

Comment:
```suggestion
```

Instead of hardcoding the namespace of the podmonitor we should add a configurable namespace selector to the pod monitor spec. This way the podmonitor can be in a different namespace than the actual pod. 

Even if we currently use the `logging` namespace it could technically be configured to be something else.